src/services/refactors/convertParamsToDestructuredObject.ts

```diff
@@ -173,6 +214,21 @@ namespace ts.refactor.convertParamsToDestructuredObject {
             const symbol = checker.getSymbolAtLocation(node);
             return symbol && getSymbolTarget(symbol, checker);
         }
+
+        function getSymbolForInterfaceSignature(node: Node): Symbol | undefined {

```

Comment:
I think the name here throws me off - it sounds like this gets the symbol from a contextual type if it comes directly from an object type.

Reply:
`contextualSymbol` sounds better to me

Reply:
Reworded.